### DISCUSSION TITLE
Synced folder gone after resume

### DISCUSSION THREAD

**Discussion by Jeansen** [vagrant-libvirt/vagrant-libvirt#1286] [Q&A]
Before I open up an issue I'd like to make sure, I am not missing something. Currently, whenever I resume from a suspended machine (Host = Debian 10, Guest = Debian 10), my shared folders (NFS) are gone. Do I have to execute extra actions to make this work?

**Comment by electrofelix** [comment]
I suspect that we should be running additional actions for the resume phase, probably prepare NFS settings, synced folders and also probably forward ports.

So no I don't think you are missing doing anything, just the resume behaviour is limited

**Reply by Jeansen** [reply]
Thanks. As it turns out, it aktually works with my setup. I just had the wrong vagrant home set (shame on me)

**Reply by Jeansen** [reply]
Anyway, this was a bit tricky, as it was not directly obvious. All VMs booted as expected but  with the wrong settings. Environment variables are always a tricky one .... ;-)

**Reply by electrofelix** [reply]
Ah, I was thinking that something might have changed since the machine was paused removing the settings added to the likes of the /etc/exports or the IP address might have changed (reboot of host), certainly there are a number of areas it could be more robust.

Thanks for confirming that the immediate problem has been resolved.
